data-raw/qc-markdowns/adult-upstream-passage-monitoring/deer-creek/extract_deer_upstream_estimates_from_raw_files.R

```diff
@@ -0,0 +1,144 @@
+# code to extract adult passage estimates from raw deer creek files
+# estimates were manually extracted into `deer_estimates_manual_extract.xlsx`
+
```

Comment:
my understanding is that this file is not being used at all but was pushed so that we have the code in case we want to pull in the files is that accurate? if so, can you clarify the comment that this file is currently not being used. i'm also considering not including it to avoid confusion (maybe we could add to the archive branch)